app/models/concerns/scripts/mode_ruby.rb

```diff
@@ -60,11 +60,17 @@ def require lib
             begin
               super lib
             rescue LoadError => ex
-              gem_command = "gem which \#{lib}"
+              raise ex unless ex.to_s.match? /cannot load such file -- \#{Regexp.escape lib}/
```

Comment:
¿No conviene poner algo más _suave_ como solo el inicio? Digo algo así:
```suggestion
              raise ex unless ex.to_s.match? /cannot load such file/i
```

Reply:
Me parece que todo este bloque quedaría mejor en una función `handle_require_exception` o similar. Ya ha crecido mucho inadvertidamente =)

Reply:
Oka, me parece bien las 2 cosas :+1: